__tests__/index.test.js

```diff
@@ -262,3 +261,37 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe('POST /api/articles/:article_id/comments', () => {
+  it('POST 201: adds comment to an article and responds with the posted comment', () => {
+    const inputBody = {
+      username: "lurker",
+      body: "Excellent work from the author"
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(inputBody)
+      .expect(201)
+      .then(({ body }) => {
+        const { newComment } = body;
+        const expected = {
+          body: "Excellent work from the author",
+          article_id: 2,
+          author: 'lurker'
+        }
+        expect(...newComment).toMatchObject(expected);
+      });
+  });
+  it('POST 400: if the inputBody is not in the correct form', () => {
+    const inputBody = {
+      username: 5,
+      body: "Excellent work"
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(inputBody)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad request")
+      });
+  });
```

Comment:
A solid happy path test ✅ Missing some error handling tests though
- 400 invalid `article_id` eg `/api/articles/katherine/comments`
- 404 non-existant article eg `/api/articles/999/comments`
- 400 missing fields in the comment body (like username or body's missing for example)
- 404 username does not exist (you already have a similar test but it's checking the datatype of the username so I'd make a separate one to be sure)